MANIFEST.in

```diff
@@ -1 +1,2 @@
+graft demo
```

Comment:
`demo/` missing in source dist. This is a fix for #77.

Reply:
Do you think it might be worth setting [check-manifest](https://pypi.org/project/check-manifest/) to avoid such problems in the future ?

Reply:
Yes that's a good idea, I'll do that.